odxtools/envdata.py

```diff
@@ -0,0 +1,66 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2022 MBition GmbH
+
+from dataclasses import dataclass
+
+from odxtools.parameters import read_parameter_from_odx
+
+from odxtools.utils import read_description_from_odx
+
+from odxtools.structures import BasicStructure
+
+from .globals import logger
+
+
+@dataclass
+class EnvData(BasicStructure):
+    """This class represents a ENV-DATA."""
```

Comment:
a slightly more elaborate docstring would be good: ENV-DATA == Enivronmental data? what is this data good for? (I know, I'm guilty of doing unhelpful docstrings myself...)

Reply:
Updated with more descriptive text